Warning CS0109 is slightly inaccurately worded

Consider this code:

```csharp
class Base
{
    private void F() {}
}
class Derived: Base
{
    new public void F() {}
}
```

Roslyn emits a warning - which it should:

>Test.cs(9,21): warning CS0109: The member 'Derived.F()' does not hide an inherited member. The new keyword is not required.

`Derived.F()` *does* hide an inherited member, it's just not an "available" (in ECMA standardese) member. In the C# 5 spec section 3.7.1.2 the reverse (when you *can* use `new`) is worded as "Contrary to hiding a name from an outer scope, hiding an accessible name from an inherited scope causes a warning to be reported".

--- a/src/Compilers/CSharp/Portable/CSharpResources.Designer.cs
+++ b/src/Compilers/CSharp/Portable/CSharpResources.Designer.cs
@@ -12984,7 +12984,7 @@ namespace Microsoft.CodeAnalysis.CSharp {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The member &apos;{0}&apos; does not hide an inherited member. The new keyword is not required..
+        ///   Looks up a localized string similar to The member &apos;{0}&apos; does not hide an accessible member. The new keyword is not required..
         /// </summary>
         internal static string WRN_NewNotRequired {
             get {

--- a/src/Compilers/CSharp/Portable/CSharpResources.resx
+++ b/src/Compilers/CSharp/Portable/CSharpResources.resx
@@ -598,7 +598,7 @@
     <value>A variable was declared with the same name as a variable in a base class. However, the new keyword was not used. This warning informs you that you should use new; the variable is declared as if new had been used in the declaration.</value>
   </data>
   <data name="WRN_NewNotRequired" xml:space="preserve">
-    <value>The member '{0}' does not hide an inherited member. The new keyword is not required.</value>
+    <value>The member '{0}' does not hide an accessible member. The new keyword is not required.</value>
   </data>
   <data name="WRN_NewNotRequired_Title" xml:space="preserve">
     <value>Member does not hide an inherited member; new keyword is not required</value>

--- a/src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOverridingAndHiding.cs
+++ b/src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOverridingAndHiding.cs
@@ -1641,7 +1641,7 @@ Derived2.Field3");
                 // (74,22): warning CS0108: 'Base2.Derived2.Type2<T>' hides inherited member 'Base2.Type2<T>'. Use the new keyword if hiding was intended.
                 //         public class Type2<T>
                 Diagnostic(ErrorCode.WRN_NewRequired, "Type2").WithArguments("Base2.Derived2.Type2<T>", "Base2.Type2<T>").WithLocation(74, 22),
-                // (99,13): warning CS0109: The member 'Derived3.Field' does not hide an inherited member. The new keyword is not required.
+                // (99,13): warning CS0109: The member 'Derived3.Field' does not hide an accessible member. The new keyword is not required.
                 //     new int Field = 2;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Field").WithArguments("Derived3.Field").WithLocation(99, 13),
                 // (101,26): warning CS0108: 'Derived3.Field2' hides inherited member 'Base3.Field2'. Use the new keyword if hiding was intended.
@@ -1810,13 +1810,13 @@ Derived2.Property1
 Base<T>.Method3()");
 
             comp.VerifyDiagnostics(
-                // (43,21): warning CS0109: The member 'Derived<U>.Method(U, U)' does not hide an inherited member. The new keyword is not required.
+                // (43,21): warning CS0109: The member 'Derived<U>.Method(U, U)' does not hide an accessible member. The new keyword is not required.
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("Derived<U>.Method(U, U)"),
                 // (47,17): warning CS0114: 'Derived<U>.Method(U, U, System.Collections.Generic.List<U>, System.Collections.Generic.Dictionary<U, U>)' hides inherited member 'Base<U>.Method(U, U, System.Collections.Generic.List<U>, System.Collections.Generic.Dictionary<U, U>)'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
                 Diagnostic(ErrorCode.WRN_NewOrOverrideExpected, "Method").WithArguments("Derived<U>.Method(U, U, System.Collections.Generic.List<U>, System.Collections.Generic.Dictionary<U, U>)", "Base<U>.Method(U, U, System.Collections.Generic.List<U>, System.Collections.Generic.Dictionary<U, U>)"),
                 // (51,17): warning CS0114: 'Derived<U>.Method<V>(V, U, System.Collections.Generic.List<V>, System.Collections.Generic.Dictionary<U, V>)' hides inherited member 'Base<U>.Method<U>(U, U, System.Collections.Generic.List<U>, System.Collections.Generic.Dictionary<U, U>)'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
                 Diagnostic(ErrorCode.WRN_NewOrOverrideExpected, "Method").WithArguments("Derived<U>.Method<V>(V, U, System.Collections.Generic.List<V>, System.Collections.Generic.Dictionary<U, V>)", "Base<U>.Method<U>(U, U, System.Collections.Generic.List<U>, System.Collections.Generic.Dictionary<U, U>)"),
-                // (55,21): warning CS0109: The member 'Derived<U>.Method<V>(V, U, System.Collections.Generic.List<V>, System.Collections.Generic.Dictionary<V, U>)' does not hide an inherited member. The new keyword is not required.
+                // (55,21): warning CS0109: The member 'Derived<U>.Method<V>(V, U, System.Collections.Generic.List<V>, System.Collections.Generic.Dictionary<V, U>)' does not hide an accessible member. The new keyword is not required.
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("Derived<U>.Method<V>(V, U, System.Collections.Generic.List<V>, System.Collections.Generic.Dictionary<V, U>)"),
                 // (64,24): warning CS0114: 'Derived<U>.Method(U)' hides inherited member 'Base<U>.Method(U)'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
                 Diagnostic(ErrorCode.WRN_NewOrOverrideExpected, "Method").WithArguments("Derived<U>.Method(U)", "Base<U>.Method(U)"),

--- a/src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs
+++ b/src/Compilers/CSharp/Test/Semantic/Semantics/InheritanceBindingTests.cs
@@ -3747,70 +3747,70 @@ class Derived : Base
 }
 ";
             CreateCompilationWithMscorlib(text).VerifyDiagnostics(
-                // (5,20): warning CS0109: The member 'C.field' does not hide an inherited member. The new keyword is not required.
+                // (5,20): warning CS0109: The member 'C.field' does not hide an accessible member. The new keyword is not required.
                 //     public new int field;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "field").WithArguments("C.field"),
-                // (6,20): warning CS0109: The member 'C.Property' does not hide an inherited member. The new keyword is not required.
+                // (6,20): warning CS0109: The member 'C.Property' does not hide an accessible member. The new keyword is not required.
                 //     public new int Property { get { return 0; } }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Property").WithArguments("C.Property"),
-                // (12,31): warning CS0109: The member 'C.Event' does not hide an inherited member. The new keyword is not required.
+                // (12,31): warning CS0109: The member 'C.Event' does not hide an accessible member. The new keyword is not required.
                 //     public new event Delegate Event;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Event").WithArguments("C.Event"),
-                // (7,26): warning CS0109: The member 'C.Interface' does not hide an inherited member. The new keyword is not required.
+                // (7,26): warning CS0109: The member 'C.Interface' does not hide an accessible member. The new keyword is not required.
                 //     public new interface Interface { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Interface").WithArguments("C.Interface"),
-                // (8,22): warning CS0109: The member 'C.Class' does not hide an inherited member. The new keyword is not required.
+                // (8,22): warning CS0109: The member 'C.Class' does not hide an accessible member. The new keyword is not required.
                 //     public new class Class { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Class").WithArguments("C.Class"),
-                // (9,23): warning CS0109: The member 'C.Struct' does not hide an inherited member. The new keyword is not required.
+                // (9,23): warning CS0109: The member 'C.Struct' does not hide an accessible member. The new keyword is not required.
                 //     public new struct Struct { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Struct").WithArguments("C.Struct"),
-                // (10,21): warning CS0109: The member 'C.Enum' does not hide an inherited member. The new keyword is not required.
+                // (10,21): warning CS0109: The member 'C.Enum' does not hide an accessible member. The new keyword is not required.
                 //     public new enum Enum { Element }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Enum").WithArguments("C.Enum"),
-                // (11,30): warning CS0109: The member 'C.Delegate' does not hide an inherited member. The new keyword is not required.
+                // (11,30): warning CS0109: The member 'C.Delegate' does not hide an accessible member. The new keyword is not required.
                 //     public new delegate void Delegate();
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Delegate").WithArguments("C.Delegate"),
-                // (13,20): warning CS0109: The member 'C.this[int]' does not hide an inherited member. The new keyword is not required.
+                // (13,20): warning CS0109: The member 'C.this[int]' does not hide an accessible member. The new keyword is not required.
                 //     public new int this[int x] { get { return 0; } }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "this").WithArguments("C.this[int]"),
-                // (19,20): warning CS0109: The member 'S.field' does not hide an inherited member. The new keyword is not required.
+                // (19,20): warning CS0109: The member 'S.field' does not hide an accessible member. The new keyword is not required.
                 //     public new int field;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "field").WithArguments("S.field"),
-                // (20,20): warning CS0109: The member 'S.Property' does not hide an inherited member. The new keyword is not required.
+                // (20,20): warning CS0109: The member 'S.Property' does not hide an accessible member. The new keyword is not required.
                 //     public new int Property { get { return 0; } }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Property").WithArguments("S.Property"),
-                // (26,31): warning CS0109: The member 'S.Event' does not hide an inherited member. The new keyword is not required.
+                // (26,31): warning CS0109: The member 'S.Event' does not hide an accessible member. The new keyword is not required.
                 //     public new event Delegate Event;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Event").WithArguments("S.Event"),
-                // (21,26): warning CS0109: The member 'S.Interface' does not hide an inherited member. The new keyword is not required.
+                // (21,26): warning CS0109: The member 'S.Interface' does not hide an accessible member. The new keyword is not required.
                 //     public new interface Interface { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Interface").WithArguments("S.Interface"),
-                // (22,22): warning CS0109: The member 'S.Class' does not hide an inherited member. The new keyword is not required.
+                // (22,22): warning CS0109: The member 'S.Class' does not hide an accessible member. The new keyword is not required.
                 //     public new class Class { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Class").WithArguments("S.Class"),
-                // (23,23): warning CS0109: The member 'S.Struct' does not hide an inherited member. The new keyword is not required.
+                // (23,23): warning CS0109: The member 'S.Struct' does not hide an accessible member. The new keyword is not required.
                 //     public new struct Struct { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Struct").WithArguments("S.Struct"),
-                // (24,21): warning CS0109: The member 'S.Enum' does not hide an inherited member. The new keyword is not required.
+                // (24,21): warning CS0109: The member 'S.Enum' does not hide an accessible member. The new keyword is not required.
                 //     public new enum Enum { Element }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Enum").WithArguments("S.Enum"),
-                // (25,30): warning CS0109: The member 'S.Delegate' does not hide an inherited member. The new keyword is not required.
+                // (25,30): warning CS0109: The member 'S.Delegate' does not hide an accessible member. The new keyword is not required.
                 //     public new delegate void Delegate();
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Delegate").WithArguments("S.Delegate"),
-                // (27,20): warning CS0109: The member 'S.this[int]' does not hide an inherited member. The new keyword is not required.
+                // (27,20): warning CS0109: The member 'S.this[int]' does not hide an accessible member. The new keyword is not required.
                 //     public new int this[int x] { get { return 0; } }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "this").WithArguments("S.this[int]"),
-                // (39,21): warning CS0109: The member 'D.Method()' does not hide an inherited member. The new keyword is not required.
+                // (39,21): warning CS0109: The member 'D.Method()' does not hide an accessible member. The new keyword is not required.
                 //     public new void Method() { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("D.Method()"),
-                // (40,20): warning CS0109: The member 'D.Property' does not hide an inherited member. The new keyword is not required.
+                // (40,20): warning CS0109: The member 'D.Property' does not hide an accessible member. The new keyword is not required.
                 //     public new int Property { get { return 0; } }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Property").WithArguments("D.Property"),
-                // (52,21): warning CS0109: The member 'Derived.Method()' does not hide an inherited member. The new keyword is not required.
+                // (52,21): warning CS0109: The member 'Derived.Method()' does not hide an accessible member. The new keyword is not required.
                 //     public new void Method() { } 
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("Derived.Method()"),
-                // (53,20): warning CS0109: The member 'Derived.Property' does not hide an inherited member. The new keyword is not required.
+                // (53,20): warning CS0109: The member 'Derived.Property' does not hide an accessible member. The new keyword is not required.
                 //     public new int Property { get { return 0; } }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Property").WithArguments("Derived.Property"),
 
@@ -5698,7 +5698,7 @@ class NS4
 }";
 
             CreateCompilationWithMscorlib(source).VerifyDiagnostics(
-                // (12,19): warning CS0109: The member 'NS1.Base2.Method' does not hide an inherited member. The new keyword is not required.
+                // (12,19): warning CS0109: The member 'NS1.Base2.Method' does not hide an accessible member. The new keyword is not required.
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("NS1.Base2.Method"),
                 // (35,30): error CS0505: 'NS2.Derived.Method<U>(System.Collections.Generic.List<int>)': cannot override because 'NS2.Base2.Method' is not a function
                 Diagnostic(ErrorCode.ERR_CantOverrideNonFunction, "Method").WithArguments("NS2.Derived.Method<U>(System.Collections.Generic.List<int>)", "NS2.Base2.Method"),
@@ -5706,7 +5706,7 @@ class NS4
                 Diagnostic(ErrorCode.ERR_CantOverrideNonProperty, "Property").WithArguments("NS2.Derived.Property", "NS2.Base2.Property"),
                 // (48,22): warning CS0108: 'NS3.Base2.Method<T>' hides inherited member 'NS3.Base<System.Collections.Generic.List<int>>.Method<U>(System.Collections.Generic.List<int>)'. Use the new keyword if hiding was intended.
                 Diagnostic(ErrorCode.WRN_NewRequired, "Method").WithArguments("NS3.Base2.Method<T>", "NS3.Base<System.Collections.Generic.List<int>>.Method<U>(System.Collections.Generic.List<int>)"),
-                // (49,26): warning CS0109: The member 'NS3.Base2.Property<T>' does not hide an inherited member. The new keyword is not required.
+                // (49,26): warning CS0109: The member 'NS3.Base2.Property<T>' does not hide an accessible member. The new keyword is not required.
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Property").WithArguments("NS3.Base2.Property<T>"),
                 // (53,30): error CS0505: 'NS3.Derived.Method<U>(System.Collections.Generic.List<int>)': cannot override because 'NS3.Base2.Method<T>' is not a function
                 Diagnostic(ErrorCode.ERR_CantOverrideNonFunction, "Method").WithArguments("NS3.Derived.Method<U>(System.Collections.Generic.List<int>)", "NS3.Base2.Method<T>"));
@@ -5788,7 +5788,7 @@ partial class NS1
                 // (13,22): warning CS0108: 'NS1.Base2.Method<T>' hides inherited member 'NS1.Base<System.Collections.Generic.List<int>>.Method<U>(System.Collections.Generic.List<int>)'. Use the new keyword if hiding was intended.
                 //         public class Method<T> { }
                 Diagnostic(ErrorCode.WRN_NewRequired, "Method").WithArguments("NS1.Base2.Method<T>", "NS1.Base<System.Collections.Generic.List<int>>.Method<U>(System.Collections.Generic.List<int>)"),
-                // (14,26): warning CS0109: The member 'NS1.Base2.Property<T>' does not hide an inherited member. The new keyword is not required.
+                // (14,26): warning CS0109: The member 'NS1.Base2.Property<T>' does not hide an accessible member. The new keyword is not required.
                 //         public new class Property<T> { }
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Property").WithArguments("NS1.Base2.Property<T>")
                 );
@@ -5821,13 +5821,13 @@ partial class Derived
 }";
 
             CreateCompilationWithMscorlib(text).VerifyDiagnostics(
-                // (4,31): warning CS0109: The member 'Base.Method()' does not hide an inherited member. The new keyword is not required.
+                // (4,31): warning CS0109: The member 'Base.Method()' does not hide an accessible member. The new keyword is not required.
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("Base.Method()"),
                 // (14,30): error CS0102: The type 'Derived' already contains a definition for 'Property'
                 Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "Property").WithArguments("Derived", "Property"),
                 // (9,25): warning CS0114: 'Derived.Method()' hides inherited member 'Base.Method()'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword.
                 Diagnostic(ErrorCode.WRN_NewOrOverrideExpected, "Method").WithArguments("Derived.Method()", "Base.Method()"),
-                // (15,32): warning CS0109: The member 'Derived.Method<T>()' does not hide an inherited member. The new keyword is not required.
+                // (15,32): warning CS0109: The member 'Derived.Method<T>()' does not hide an accessible member. The new keyword is not required.
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Method").WithArguments("Derived.Method<T>()"));
         }
 
@@ -5862,10 +5862,10 @@ class Derived : Base
 }";
 
             CreateCompilationWithMscorlib(text).VerifyDiagnostics(
-                // (12,20): warning CS0109: The member 'Derived.MethOd' does not hide an inherited member. The new keyword is not required.
+                // (12,20): warning CS0109: The member 'Derived.MethOd' does not hide an accessible member. The new keyword is not required.
                 //     public new int MethOd = 2, Method = 3, METhod = 4;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "MethOd").WithArguments("Derived.MethOd"),
-                // (12,44): warning CS0109: The member 'Derived.METhod' does not hide an inherited member. The new keyword is not required.
+                // (12,44): warning CS0109: The member 'Derived.METhod' does not hide an accessible member. The new keyword is not required.
                 //     public new int MethOd = 2, Method = 3, METhod = 4;
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "METhod").WithArguments("Derived.METhod"),
                 // (19,22): warning CS0108: 'Derived.Base2.Type' hides inherited member 'Base.Type'. Use the new keyword if hiding was intended.

--- a/src/Compilers/CSharp/Test/Symbol/Symbols/InterfaceOverriddenOrHiddenMembersTests.cs
+++ b/src/Compilers/CSharp/Test/Symbol/Symbols/InterfaceOverriddenOrHiddenMembersTests.cs
@@ -38,7 +38,7 @@ interface I0 : I1, I2
 ";
 
             CreateCompilationWithMscorlib(source).VerifyDiagnostics(
-                // (13,14): warning CS0109: The member 'I0.foo(int)' does not hide an inherited member. The new keyword is not required.
+                // (13,14): warning CS0109: The member 'I0.foo(int)' does not hide an accessible member. The new keyword is not required.
                 //     new void foo(int x);
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "foo").WithArguments("I0.foo(int)"));
         }

--- a/src/Compilers/CSharp/Test/Symbol/Symbols/OverriddenOrHiddenMembersTests.cs
+++ b/src/Compilers/CSharp/Test/Symbol/Symbols/OverriddenOrHiddenMembersTests.cs
@@ -3508,58 +3508,58 @@ public class D8 : B
 ";
 
             CreateCompilationWithMscorlib(source).VerifyDiagnostics(
-                // (23,22): warning CS0109: The member 'D1.Finalize' does not hide an inherited member. The new keyword is not required.
+                // (23,22): warning CS0109: The member 'D1.Finalize' does not hide an accessible member. The new keyword is not required.
                 //     new public class Finalize { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D1.Finalize"),
-                // (21,22): warning CS0109: The member 'D1.op_UnaryPlus' does not hide an inherited member. The new keyword is not required.
+                // (21,22): warning CS0109: The member 'D1.op_UnaryPlus' does not hide an accessible member. The new keyword is not required.
                 //     new public class op_UnaryPlus { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_UnaryPlus").WithArguments("D1.op_UnaryPlus"),
-                // (22,22): warning CS0109: The member 'D1.op_Explicit' does not hide an inherited member. The new keyword is not required.
+                // (22,22): warning CS0109: The member 'D1.op_Explicit' does not hide an accessible member. The new keyword is not required.
                 //     new public class op_Explicit { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_Explicit").WithArguments("D1.op_Explicit"),
-                // (58,30): warning CS0109: The member 'D6.Finalize' does not hide an inherited member. The new keyword is not required.
+                // (58,30): warning CS0109: The member 'D6.Finalize' does not hide an accessible member. The new keyword is not required.
                 //     new public delegate void Finalize(); //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D6.Finalize"),
-                // (56,30): warning CS0109: The member 'D6.op_UnaryPlus' does not hide an inherited member. The new keyword is not required.
+                // (56,30): warning CS0109: The member 'D6.op_UnaryPlus' does not hide an accessible member. The new keyword is not required.
                 //     new public delegate void op_UnaryPlus(); //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_UnaryPlus").WithArguments("D6.op_UnaryPlus"),
-                // (57,30): warning CS0109: The member 'D6.op_Explicit' does not hide an inherited member. The new keyword is not required.
+                // (57,30): warning CS0109: The member 'D6.op_Explicit' does not hide an accessible member. The new keyword is not required.
                 //     new public delegate void op_Explicit(); //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_Explicit").WithArguments("D6.op_Explicit"),
-                // (37,20): warning CS0109: The member 'D3.Finalize' does not hide an inherited member. The new keyword is not required.
+                // (37,20): warning CS0109: The member 'D3.Finalize' does not hide an accessible member. The new keyword is not required.
                 //     new public int Finalize = 1; //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D3.Finalize"),
-                // (51,36): warning CS0109: The member 'D5.Finalize' does not hide an inherited member. The new keyword is not required.
+                // (51,36): warning CS0109: The member 'D5.Finalize' does not hide an accessible member. The new keyword is not required.
                 //     new public event System.Action Finalize { add { } remove { } } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D5.Finalize"),
-                // (71,27): warning CS0109: The member 'D8.op_Explicit(B)' does not hide an inherited member. The new keyword is not required.
+                // (71,27): warning CS0109: The member 'D8.op_Explicit(B)' does not hide an accessible member. The new keyword is not required.
                 //     new public static int op_Explicit(B b) { return 0; } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_Explicit").WithArguments("D8.op_Explicit(B)"),
-                // (72,28): warning CS0109: The member 'D8.Finalize()' does not hide an inherited member. The new keyword is not required.
+                // (72,28): warning CS0109: The member 'D8.Finalize()' does not hide an accessible member. The new keyword is not required.
                 //     new public static void Finalize() { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D8.Finalize()"),
-                // (70,27): warning CS0109: The member 'D8.op_UnaryPlus(B)' does not hide an inherited member. The new keyword is not required.
+                // (70,27): warning CS0109: The member 'D8.op_UnaryPlus(B)' does not hide an accessible member. The new keyword is not required.
                 //     new public static int op_UnaryPlus(B b) { return 0; } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_UnaryPlus").WithArguments("D8.op_UnaryPlus(B)"),
-                // (44,36): warning CS0109: The member 'D4.Finalize' does not hide an inherited member. The new keyword is not required.
+                // (44,36): warning CS0109: The member 'D4.Finalize' does not hide an accessible member. The new keyword is not required.
                 //     new public event System.Action Finalize; //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D4.Finalize"),
-                // (64,21): warning CS0109: The member 'D7.op_Explicit(B)' does not hide an inherited member. The new keyword is not required.
+                // (64,21): warning CS0109: The member 'D7.op_Explicit(B)' does not hide an accessible member. The new keyword is not required.
                 //     new public void op_Explicit(B b) { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_Explicit").WithArguments("D7.op_Explicit(B)"),
-                // (65,21): warning CS0109: The member 'D7.Finalize()' does not hide an inherited member. The new keyword is not required.
+                // (65,21): warning CS0109: The member 'D7.Finalize()' does not hide an accessible member. The new keyword is not required.
                 //     new public void Finalize() { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D7.Finalize()"),
-                // (63,21): warning CS0109: The member 'D7.op_UnaryPlus(B)' does not hide an inherited member. The new keyword is not required.
+                // (63,21): warning CS0109: The member 'D7.op_UnaryPlus(B)' does not hide an accessible member. The new keyword is not required.
                 //     new public void op_UnaryPlus(B b) { } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_UnaryPlus").WithArguments("D7.op_UnaryPlus(B)"),
-                // (29,20): warning CS0109: The member 'D2.op_Explicit' does not hide an inherited member. The new keyword is not required.
+                // (29,20): warning CS0109: The member 'D2.op_Explicit' does not hide an accessible member. The new keyword is not required.
                 //     new public int op_Explicit { get; set; } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_Explicit").WithArguments("D2.op_Explicit"),
-                // (30,20): warning CS0109: The member 'D2.Finalize' does not hide an inherited member. The new keyword is not required.
+                // (30,20): warning CS0109: The member 'D2.Finalize' does not hide an accessible member. The new keyword is not required.
                 //     new public int Finalize { get; set; } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "Finalize").WithArguments("D2.Finalize"),
-                // (28,20): warning CS0109: The member 'D2.op_UnaryPlus' does not hide an inherited member. The new keyword is not required.
+                // (28,20): warning CS0109: The member 'D2.op_UnaryPlus' does not hide an accessible member. The new keyword is not required.
                 //     new public int op_UnaryPlus { get; set; } //CS0109
                 Diagnostic(ErrorCode.WRN_NewNotRequired, "op_UnaryPlus").WithArguments("D2.op_UnaryPlus"),
 

